packages/eslint-plugin-orbit-components/src/rules/noDeprecatedToken.ts

```diff
@@ -0,0 +1,96 @@
+import * as t from "@babel/types";
+import type { Rule } from "eslint";
+/*
```

Comment:
cc @silvenon

Reply:
For me it totally makes sense that `@kiwicom/orbit-design-tokens` is a dependency of `eslint-plugin-orbit-components` because of this rule, at least for those versions of the ESLint plugin which will have this rule. What would you gain by `@kiwicom/orbit-components` re-exporting it…? I'm wondering what is the dilemma.

Reply:
I was just not sure if we want to add another dependency to the whole cycle. `@kiwicom/orbit-design-tokens` are direct dependency of `@kiwicom/orbit-components`, but since it won't be peer dependency (for components and the eslint plugin) it makes sense to simply add it 🙂 Thanks for your input @silvenon